cl/search/migrations/0017_remove_bankruptcyinformation_update_or_delete_snapshot_update_and_more.sql

```diff
@@ -0,0 +1,634 @@
+BEGIN;
+--
+-- Remove trigger update_or_delete_snapshot_update from model bankruptcyinformation
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_17e86 ON "search_bankruptcyinformation";
+--
+-- Remove trigger update_or_delete_snapshot_update from model claim
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_bb32f ON "search_claim";
+--
+-- Remove trigger update_or_delete_snapshot_update from model claimhistory
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_137a5 ON "search_claimhistory";
+--
+-- Remove trigger update_or_delete_snapshot_update from model court
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_c94ab ON "search_court";
+--
+-- Remove trigger update_or_delete_snapshot_update from model docket
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_7e039 ON "search_docket";
+--
+-- Remove trigger update_or_delete_snapshot_update from model docketentry
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_46e1e ON "search_docketentry";
+--
+-- Remove trigger update_or_delete_snapshot_update from model opinion
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_67ecd ON "search_opinion";
+--
+-- Remove trigger update_or_delete_snapshot_update from model opinioncluster
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_6a181 ON "search_opinioncluster";
+--
+-- Remove trigger update_or_delete_snapshot_update from model originatingcourtinformation
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_49538 ON "search_originatingcourtinformation";
+--
+-- Remove trigger update_or_delete_snapshot_update from model recapdocument
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_8a108 ON "search_recapdocument";
+--
+-- Remove trigger update_or_delete_snapshot_update from model tag
+--
+DROP TRIGGER IF EXISTS pgtrigger_update_or_delete_snapshot_update_c9dd9 ON "search_tag";
+--
+-- Rename unnamed index for ('volume', 'reporter') on citation to search_cita_volume_464334_idx
+--
+-- (no-op)
```

Comment:
Well this is a bit odd. What's an unnamed index? In prod, it's named `search_citation_volume_251bc1d270a8abee_idx`

Reply:
Looking in Django's issue tracker, an "unnamed index" is what they call the ones generated by `index_together`. It seems like they struggled to generate these migration files since finding old indexes was hard and they settled on doing noops some of the time (which isn't awful, the DB performs the same).

Reply:
Anyway, this is the last thing before merging is figuring out what to do with these last five indexes.

Reply:
so do we need to actually do anything for these no-op migrations or should it work fine?